vspace/src/main/java/edu/asu/diging/vspace/core/data/SlideExternalLinkRepository.java

```diff
@@ -0,0 +1,21 @@
+package edu.asu.diging.vspace.core.data;
+
+import java.util.List;
+
+import org.javers.spring.annotation.JaversSpringDataAuditable;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.PagingAndSortingRepository;
+import org.springframework.stereotype.Repository;
+
+import edu.asu.diging.vspace.core.model.IExternalLinkSlide;
+import edu.asu.diging.vspace.core.model.impl.ExternalLinkSlide;
+
+@Repository
+@JaversSpringDataAuditable
+public interface SlideExternalLinkRepository extends PagingAndSortingRepository<ExternalLinkSlide, String> {
+
+    @Query("SELECT d FROM ExternalLinkSlide d WHERE d.slide.id = ?1")
```

Comment:
this is a simple query and it should be possible to use the method naming convention to create the query (something like `getBySlide` or something like that)